region/crateria/west/Gauntlet Energy Tank Room.json

```diff
@@ -203,6 +233,151 @@
+    },
+    {
+      "link": [1, 1],
+      "name": "G-Mode Morph, Bomb the Blocks",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "any",
+          "morphed": true
+        }
+      },
+      "requires": [
+        "Bombs"
+      ],
+      "clearsObstacles": ["C"],
+      "note": "Carefully kill the Zebbo, then place two bombs against the right wall to break the left half of the wall to slightly increase the length of the runway."
+    },
+    {
+      "link": [1, 1],
+      "name": "Gauntlet E-Tank R-Mode Blue Suit",
+      "notable": true,
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        "canBePatient",
+        {"or": [
+          {"canShineCharge": {
+            "usedTiles": 14,
+            "openEnd": 0,
+            "steepUpTiles": 1,
+            "steepDownTiles": 1
+          }},
+          {"and": [
+            {"canShineCharge": {
+              "usedTiles": 15,
+              "openEnd": 0,
+              "steepUpTiles": 1,
+              "steepDownTiles": 1
+            }},
+            {"doorUnlockedAtNode": 1}
+          ]},
+          {"and": [
+            {"canShineCharge": {
+              "usedTiles": 18,
+              "openEnd": 1,
+              "steepUpTiles": 1,
+              "steepDownTiles": 1
+            }},
+            "h_canDestroyBombWalls"
+          ]},
+          {"and": [
+            {"canShineCharge": {
+              "usedTiles": 19,
+              "openEnd": 1,
+              "steepUpTiles": 1,
+              "steepDownTiles": 1
+            }},
+            "h_canDestroyBombWalls",
+            {"doorUnlockedAtNode": 1}
+          ]}
+        ]}
+      ],
+      "clearsObstacles": ["B", "D"],
+      "note":  [
+        "Use R-Mode to get a blue suit to cross the room and get the item, or to leave the right shinecharged.",
+        "In R-Mode, fill your Reserves at the farm. Damage down again so that Samus is within 1 Zebbo hit from triggering Reserves.",
+        "Gain a shinecharge in-room without gaining more energy. It is possible to time the despawn of a drop with the acid if no ammo is owned.",
+        "Shinespark vertically, such that a Zebbo can hit Samus in the wind-up to cause a reserve trigger, giving a blue suit.",
+        "It is then fine to farm the Zebbos again."
+      ]
+    },
+    {
+      "link": [1, 3],
+      "name": "Blue Suit",
+      "requires": [
+        {"obstaclesCleared": ["D"]},
+        "Morph",
+        {"or": [
+          "canTrickyJump",
+          {"acidFrames": 20}
+        ]}
+      ],
+      "clearsObstacles": ["A"],
+      "note": [
+        "Carefully walk through the room. It is possible but tricky to do so damageless, but it is always possible to go back and farm if needed.",
+        "While in the morph tunnel, simply unmorph to break the bomb blocks overhead."
+      ]
+    },
+    {
+      "link": [1, 4],
```

Comment:
I think this is supposed to be [1,1] and then uses a obstaclesCleared[B] strat to reach 4